src/main/scala/tech/sourced/gemini/Gemini.scala

```diff
@@ -36,21 +36,17 @@ class Gemini(session: SparkSession, keyspace: String = Gemini.defautKeyspace) {
   }
 
   def hashForRepos(repos: DataFrame): DataFrame = {
```

Comment:
I think that now you removed the assignation you can also remove the `{` and `}` .

Reply:
you are absolutely right or how people say in Vietnam "Đúng rồi"!
Cảm ơn nha! Fixed.